src/inmanta/util/dict_path.py

```diff
@@ -593,6 +771,26 @@ def get_elements(self, container: object) -> list[object]:
 
         return containers
 
+    def resolve_wild_cards(self, container: object) -> Sequence["WildComposedPath"]:
```

Comment:
This could be considered a tree if I understand correctly, where very wildcard creates/can create a branching point?


(not actionable, just for me to understand)

Reply:
I would rather say that every wild path is a branching point, but yes, it can be seen as a tree